src/osd/PG.cc

```diff
@@ -5975,6 +5978,18 @@ boost::statechart::result PG::RecoveryState::Peering::react(const QueryState& q)
   }
   q.f->close_section();
 
+  if (history_les_bound) {
+    q.f->open_object_section("peering_blocked_by_options");
```

Comment:
On Wed, 16 Mar 2016, David Zafman wrote:
> 
> In src/osd/PG.cc:
> 
> > @@ -5975,6 +5978,18 @@ boost::statechart::result PG::RecoveryState::Peering::react(const QueryState& q)
> >    }
> >    q.f->close_section();
> >  
> > +  if (history_les_bound) {
> > +    q.f->open_object_section("peering_blocked_by_options");
> 
> Even though you warn about it, I'm nervous calling this "peering_blocked_by_options." Someone might assume that they should leave
> osd_find_best_info_ignore_history_les permanently set to true.
> 
> I wish the option were called "osd_find_best_info_ignore_history_les_and_lose_data_i_really_really_mean_it" or invert it as
> "osd_find_best_info_check_data_integrity_history_les" defaulting to true.

Yeah... we could not mention the option, maybe?  Although the field that 
mentions the option also mentioned that you will likely lose data.  But